vignettes/articles/crane.Rmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "Introduction to the {crane} package"
+author: Emily de la Rua
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## What is {crane}?
+
+The {crane} package was created to supplement the {gtsummary} package by providing additional functionality for creating clinical trial tables used specifically within the pharmaceutical industry. Since gtsummary was created as a very general framework to build tables, we want to avoid cluttering that package with pharma-specific functions that are only used by a small subset of {gtsummary} users. Instead, this pharma-specific functionality is added to the {crane} package for pharmaceutical data scientists to use. 
+
+This package includes several wrapper functions for pre-existing {gtsummary} functions which make it easier to apply common pharma-specific modifications to basic table formats (for example, demographics tables which use certain variables and formatting conventions as a default). 
+
+When the crane package is loaded into your R session, a Roche gtsummary theme is loaded by default which automatically applies Roche-specific formatting to any subsequently generated gtsummary tables. This includes applying standard settings used by the rtables & tern packages, such as using a mono-spaced font for `gt` tables and specific p-value formatting. See `gtsummary::theme_gtsummary_roche()` documentation for details.
+
+#### Where should I add new functions?
+
+- If the function might be useful to R programmers outside of the pharmaceutical industry, add it to **{gtsummary}**.
+- If the function is used specifically to adhere to pharmaceutical standards and/or to create FDA tables, TLG catalog tables, etc., add it to **{crane}**.
+
+## {crane} Functionality
+
+To keep the templates in the {tlg.templates} package simple, if a template cannot be easily constructed with the functions already available in {gtsummary} and {crane} (e.g. many functions or complex modifications are needed) consider adding a function to {crane} to assist.
```

Comment:
Let's not mention tlg.template because crane is external and tlg.templates is not

Reply:
Updated